packages/babel-plugin/src/utils/ast-builders.tsx

```diff
@@ -46,12 +46,16 @@ const hoistSheet = (sheet: string, meta: Metadata): t.Identifier => {
   }
 
   const sheetIdentifier = meta.parentPath.scope.generateUidIdentifier('');
-  const parent = meta.parentPath.findParent((path) => path.isProgram()).get('body') as NodePath[];
-  const path = parent.filter((path) => !path.isImportDeclaration())[0];
-
-  path.insertBefore(
-    t.variableDeclaration('const', [t.variableDeclarator(sheetIdentifier, t.stringLiteral(sheet))])
-  );
+  const parent = meta.parentPath.findParent((path) => path.isProgram());
+  const parentBody = parent && (parent.get('body') as NodePath[]);
```

Comment:
So all of these started breaking just after patch/minor bump ups? 😨

Reply:
yeah